test/tests/open-liberty-hello-world/run.sh

```diff
@@ -5,8 +5,8 @@ dir="$(dirname "$(readlink -f "$BASH_SOURCE")")"
 
 image="$1"
 
-# since we have curl in the liberty image, we'll use that
-clientImage="$1"
+# Use the alpine image since it is small and has wget in it that we can use
+clientImage="alpine:latest"
```

Comment:
I'd recommend updating this to a specific version (like `alpine:3.9`), but that can be a separate PR (I'll have a build test in a second). :+1: